ckanext/qgov/common/intercepts.py

```diff
@@ -48,53 +48,20 @@
 STORAGE_DOWNLOAD = StorageController.file
 RESOURCE_DOWNLOAD = PackageController.resource_download
 
-ALLOWED_EXTENSIONS = [
-    'accdb',
-    'csv',
-    'doc',
-    'docx',
-    'esri',
-    'fgdb',
-    'gdb',
-    'geojson',
-    'geotiff',
-    'gpkg',
-    'gpx',
-    'html',
-    'jp2',
-    'jpeg',
-    'jpg',
-    'json',
-    'kml',
-    'kmz',
-    'mtl',
-    'n3',
-    'obj',
-    'pdf',
-    'png',
-    'ppt',
-    'pptx',
-    'rdf',
-    'rtf',
-    'shp',
-    'sparql',
-    'tab',
-    'tif',
-    'tiff',
-    'ttf',
-    'topojson',
-    'txt',
-    'wfs',
-    'wmts',
-    'xls',
-    'xlsx',
-    'xml',
-    'zip'
-]
+file_mime_config = json.load(open(
+    os.path.join(os.path.dirname(__file__), 'resources', 'resource_types.json')
```

Comment:
could we rename the file to resource_types_allowed.json ?

Reply:
It's not just allowed types, it's also a list of extra entries to be added to the 'mimetypes' library.